packages/mobile/src/utils/reachability.ts

```diff
@@ -65,7 +64,6 @@ const updateReachability = async (netInfoState: NetInfoState) => {
     // Supercede the setUnreachable debounce if necessary
     setUnreachable(false)
     dispatch(reachabilityActions.setReachable())
-    storeContext.apiClient.setIsReachable(true)
```

Comment:
So this wasn't working? Or it just didn't always get called?

Reply:
oh wow epic find! this might fix another issue we were seeing with download errors when going offline

Reply:
and also possibly the empty favorites lineup while starting offline

Reply:
So this was definitely causing an issue when manually setting the reachability via an action. It also seems to fix the issue with favorites pagination but I'm not entirely certain how the reachability was getting out of sync in that case, because this is the only place we dispatch the action

Gonna dig a bit more before merging this